### PR TITLE
DOC: README.rst: Installation (conda install -c conda conda-build)

### PR DESCRIPTION
**Pull request by westurner**

### CODE REVIEW

**Comment by asmeurer** [comment]
We haven't been maintaining the conda channel on binstar. I would recommend removing that part.

**Comment by westurner** [comment]
I have removed that part. Barring any other changes this should be ready to merge.

File: README.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 ===========
 
+Installation
+--------------
+.. code:: bash
+
+    # Display information about current conda install
+    conda info
+
+    # Install conda-build in the current env
```
Comment (asmeurer) [inline README.rst]: conda-build can only be installed into the root environment. Perhaps the install command should be `conda install -n root conda-build`.

**Comment by asmeurer** [comment]
Sorry, just noticed another issue. Other than that this should be good.

**Comment by westurner** [comment]
Changed to ``conda install -n root conda-build``.

It may be helpful to clarify this in http://conda.pydata.org/docs/build_tutorials/pkgs.html#install-conda-and-conda-build as well

**Comment by asmeurer** [comment]
CC @kerrywatson1

**Comment by westurner** [comment]
(Thanks!)

**Comment by westurner** [comment]
Note that I linked to \#installation in https://github.com/conda/conda-recipes/pull/290

**Pull request merged**